lua/plenary/window/border.lua

```diff
@@ -28,35 +69,42 @@ function Border._create_lines(content_win_options, border_win_options)
   local topleft = (left_enabled and border_win_options.topleft) or ""
   local topright = (right_enabled and border_win_options.topright) or ""
 
+  local titles
+  if type(border_win_options.title) == "string" then
+    titles = { { "top-mid", border_win_options.title } }
+  elseif type(border_win_options.title) == "table" and type(border_win_options.title[1]) == "table" then
+    titles = border_win_options.title
+  elseif type(border_win_options.title) == "table" and type(border_win_options.title[1]) == "string" then
+    titles = { border_win_options.title }
+  elseif not border_win_options.title then
+    titles = {}
+  else
+    error("Invalid option for `border_win_options.title`: " .. tostring(border_win_options.title))
```

Comment:
That's the only option being validated right now. I'd remove it to simplify and rather add `vim.validate` at top of `create` if for all flags you want to. If `string` behavior is kept as is, then just removing this should also be fine, at least up until docgen. Thoughts?

Reply:
I don't think it is necessary to validate this at this point, we can always add validation in later if needed.